.travis.yml

```diff
@@ -8,7 +8,7 @@ env:
     - PYTHON=Conda
 julia:
     - 1.0
-    - 1.4
+    - 1.5
```

Comment:
Why not just use `1`?

Reply:
yeah, I keep forgetting that, I only recently realized that putting `1` gives latest `1`.  Let's see what the test result is and I'll put it in.